src/compute/src/compute_state.rs

```diff
@@ -80,21 +75,63 @@ pub struct ComputeState {
     /// History of commands received by this workers and all its peers.
     pub command_history: ComputeCommandHistory<UIntGauge>,
     /// Max size in bytes of any result.
-    pub max_result_size: u32,
+    max_result_size: u32,
```

Comment:
Why isn't this `pub` anymore? Same question for `tracing_handle`.

Reply:
Simply because it doesn't need to be. It needed to be `pub` before only because `ComputeState` didn't have a constructor but was instantiated in a different module. I'm defaulting to making things as private as possible, but no strong feelings about keeping this `pub` instead if there is a reason to do so?

Reply:
Sounds good. It seems in the long run, all fields should be private.